database/other/text-formatting.json

```diff
@@ -0,0 +1,7 @@
+[{
+    "name": "Text Tune",
+    "description": "Text tune is the ultimate solution for all the text formatting needs!The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide with synonyms and antonyms for the word given by the user. If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful.",
```

Comment:
```suggestion
    "description": "Text tune is the ultimate solution for all the text formatting needs! The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide synonyms and antonyms for the word given by the user.",
```

> If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful.

<details><summary>Why remove this?</summary>
This sentence isn't required, it just reference to the "synonyms and antonyms" feature again.



</p>
</details>